moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -258,6 +258,7 @@ bool MoveItConfigData::outputSetupAssistantFile(const std::string& file_path)
   emitter << YAML::Value << YAML::BeginMap;
   emitter << YAML::Key << "package" << YAML::Value << urdf_pkg_name_;
   emitter << YAML::Key << "relative_path" << YAML::Value << urdf_pkg_relative_path_;
+  emitter << YAML::Key << "use_jade_xacro" << YAML::Value << urdf_requires_jade_xacro_;
```

Comment:
That occured to me as well.

But I couldn't really make up my mind as I don't see any obvious problems with including it even if it is `false`. Only thing I could come up with was it might 'invite' users to change it to `true` after the package is generated, expecting the rest of the infrastructure to automagically cope with Jade+ xacros as well - which is obviously not going to happen.

tl;dr: I don't really mind either way, so if anyone has a strong preference for not serialising in case the property is `false`, I can change it.